hw/dv/sv/i2c_agent/i2c_agent.core

```diff
@@ -21,6 +21,7 @@ filesets:
       - i2c_agent.sv: {is_include_file: true}
       - seq_lib/i2c_seq_list.sv: {is_include_file: true}
       - seq_lib/i2c_base_seq.sv: {is_include_file: true}
+      - seq_lib/i2c_cp_wdata_to_rdata_seq.sv: {is_include_file: true}
```

Comment:
This is not quite memory type mode.
Test sends burst of write request followed by read request. Write might be hit the same address multiple times.
I move to array of q to sequence instead of sequencer and reuse req_analysis_port to remove all 'lb_port[] -> fifo' connections.